ci/plugins/mzconduct/hooks/command

```diff
@@ -17,6 +17,12 @@ mzconduct() {
 
 test_name="$BUILDKITE_PLUGIN_MZCONDUCT_TEST"
 workflow="$BUILDKITE_PLUGIN_MZCONDUCT_WORKFLOW"
+# Note that there is a \r that seems to be emitted by docker that we need to strip
+compose_file="$(bin/ci-builder run stable bin/mzconduct show dir "$test_name" | tr -d $'\r')/mzcompose.yml"
```

Comment:
This seems sketch. I suspect we need to strip the —tty flag when stdout is not a terminal or something.

Reply:
I copied this from the `pre-exit` hook

Reply:
I think I've got it fixed in https://github.com/MaterializeInc/materialize/pull/3723

Reply:
Got this fixed, so you should be able to drop the `tr`!